static/app/components/group/releaseChart.tsx

```diff
@@ -199,4 +198,9 @@ function GroupReleaseChart(props: Props) {
   );
 }
 
+const EventNumber = styled('div')`
+  margin-top: -5px;
```

Comment:
This is a gross hack and we should think of another way to do this for Big Numbers in the sidenav. We already do something similar on the Transaction Summary page so it’s likely we’ll need to use something like this again and again.